### PR TITLE
Add Spectro-K8s-dashboard

### PR DESCRIPTION
**Pull request by ritawatson**
This PR ...

- Documents the new **Spectro-Kubernetes-Dashboard** pack. ([PEM-384](https://spectrocloud.atlassian.net/browse/PEM-384)) 
A list of supported clouds is included. ([PEM-984](https://spectrocloud.atlassian.net/browse/PEM-984))

@ravi-k8, please verify the list of supported clouds is accurate as QA testing progresses.

- Adds a new filter category labeled **Integrations**. Spectro Kubernetes Dashboard appears in both the **Integrations** and **Monitoring** categories. -->**Filter category removed.**
- Adds a note in **K8s Dashboard** doc about the **Connect** button and cross references the Spectro K8s Dashboard. ([PAC-281](https://spectrocloud.atlassian.net/browse/PAC-281)) Existing content is also clarified a bit and revised to use our internal style - refactoring to happen post 3.2.
- Changes 'Pack Integrations' in the nav pane to 'Packs List'.
- Adds a link in the Packs List overview to the "Stand-alone and Integrated Packs" doc. --> **Link to sub-doc in index page removed and sub-doc removed**.

[Spectro K8s Dashboard Preview](https://deploy-preview-1106--docs-spectrocloud.netlify.app/integrations/spectro-k8s-dashboard)
[K8s Dashboard Preview](https://deploy-preview-1106--docs-spectrocloud.netlify.app/integrations/kubernetes-dashboard)

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1106#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by karl-cardenas-coding** [comment]
@ritawatson Take out the tab in this case. No point in having a tab when it's only 1 available option.

File: content/docs/06-integrations/00-spectro-k8s-dashboard.md
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-spectro-k8s-dashboard.md:8]:
```suggestion
category: ['Spectro Integrations', 'monitoring']
```
We need both categories as in the UI today, users will find it through the monitoring category.

File: content/docs/06-integrations/00-spectro-k8s-dashboard.md
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-spectro-k8s-dashboard.md:3]:
```suggestion
metaTitle: 'Spectro Cloud's Integrated Kubernetes Dashboard'
```
For SEO purposes.

File: content/docs/06-integrations/00-spectro-k8s-dashboard.md
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-spectro-k8s-dashboard.md:4]:
```suggestion
metaDescription: 'The Integrated Kubernetes Dashboard Monitoring pack in Spectro Cloud reduces the complexity of standing up the Kubernetes dashboard for a cluster. '
```

File: content/docs/06-integrations/00-spectro-k8s-dashboard.md
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-spectro-k8s-dashboard.md:83]: Let's explain what using Palette as a value actually means. When you select Palette it means Palette becomes the identity provider, so any user that has a Palette account in the tenant and the proper permissions to view and access the respective project's resources will be able to log into the dashboard.

File: content/docs/06-integrations/00-spectro-k8s-dashboard.md
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-spectro-k8s-dashboard.md:85]: Let's link to the Palette SSO page. https://docs.spectrocloud.com/user-management/saml-sso

File: content/docs/06-integrations/00-spectro-k8s-dashboard.md
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-spectro-k8s-dashboard.md:91]: These parameters are already in the Kubernetes pack YAML definition. The user just needs to uncomment them, so there is no need to copy them.

File: content/docs/06-integrations/00-spectro-k8s-dashboard.md
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-spectro-k8s-dashboard.md:94]: This configuration is only applicable to EKS. Ideally, we should showcase the same for Azure and GCP. This is where Ravi suggested we ask the Q/A team. Arvind and Resma are two names that come to mind.

**Review by karl-cardenas-coding — CHANGES_REQUESTED** [review]

**Comment by spectro-prow** [comment]
@ritawatson: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | cd45a86c04bba37255b20eb706cf098eae022194 | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1106/librarium_build/1621661551847018496) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: content/docs/06-integrations/00-spectro-k8s-dashboard.md
Comment (ritawatson) [reply content/docs/06-integrations/00-spectro-k8s-dashboard.md:20]: Revised this description.

File: content/docs/06-integrations/00-spectro-k8s-dashboard.md
Comment (ritawatson) [reply content/docs/06-integrations/00-spectro-k8s-dashboard.md:83]: Added.

File: content/docs/06-integrations/00-spectro-k8s-dashboard.md
Comment (ritawatson) [reply content/docs/06-integrations/00-spectro-k8s-dashboard.md:85]: Added link.

File: content/docs/06-integrations/00-spectro-k8s-dashboard.md
Comment (ritawatson) [reply content/docs/06-integrations/00-spectro-k8s-dashboard.md:91]: @karl-cardenas-coding I don't see the "OIDC related config" in the config file. 

@ravi-k8 @shantanubansal Can you please confirm? If a user changes the default value to 'Inherit from tenant', do they have to add this "OIDC related config" block?

File: content/docs/06-integrations/00-spectro-k8s-dashboard.md
Comment (ritawatson) [reply content/docs/06-integrations/00-spectro-k8s-dashboard.md:94]: @ravi-k8 @shantanubansal Do either of you have the OIDC configuration for Azure and GCP to include in the doc?

**Comment by spectro-prow** [comment]
@ritawatson: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | 44c4ab1b85128058acd12f55a8f3d00d92f31fa6 | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1106/librarium_build/1621675266071859200) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by spectro-prow** [comment]
@ritawatson: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | 526f04cc1c428acde249a7d57aa67f1384cd7f29 | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1106/librarium_build/1621676020941721600) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: content/docs/06-integrations/00-spectro-k8s-dashboard.md
Comment (ravi-k8) [reply content/docs/06-integrations/00-spectro-k8s-dashboard.md:93]: @ritawatson I think the label should be "Inherit from Org"
If user selects "Inherit from Org" then backend would take care of adding client config block at run time. User is not required to add additional configuration.

File: content/docs/06-integrations/00-spectro-k8s-dashboard.md
Comment (shantanubansal) [reply content/docs/06-integrations/00-spectro-k8s-dashboard.md:96]: @ritawatson since we are providing a smooth and enhanced experience to user, as a part of that configuration is not directly visible to the user. But still In documentation we can give some configuration on How to Enable OIDC through our pack for different clouds.

@ritawatson @karl-cardenas-coding  maybe with the help of content team and testing team configuration also can be summarized in documentation for all the cloud types we support.

**Comment by spectro-prow** [comment]
@ritawatson: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | 526f04cc1c428acde249a7d57aa67f1384cd7f29 | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1106/librarium_build/1622641421544591360) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1106#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-2/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
@ritawatson: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | 781526ad350b23802dc4e2674355a918b3aeb035 | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1106/librarium_build/1622662688209375232) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: content/docs/06-integrations/00-spectro-k8s-dashboard.md
Comment (ritawatson) [reply content/docs/06-integrations/00-spectro-k8s-dashboard.md:103]: @ravi-k8 Has the UI label been changed? I'm going by the demo that was given. The UI at the time said 'Inherit from Tenant'.

File: content/docs/06-integrations/00-spectro-k8s-dashboard.md
Comment (ritawatson) [reply content/docs/06-integrations/00-spectro-k8s-dashboard.md:106]: @shantanubansal I need your help/information about how users can enable OIDC through the Spectro K8s Dashboard for different clouds. Can you please provide some details?

File: content/docs/06-integrations/00-spectro-k8s-dashboard.md
Comment (ritawatson) [inline content/docs/06-integrations/00-spectro-k8s-dashboard.md:227]: @MGamya @arvind-sys
The above command is in the original documentation. 
- Why would users run the command? 
- What does the command do?

File: content/docs/06-integrations/00-spectro-k8s-dashboard.md
Comment (ritawatson) [inline content/docs/06-integrations/00-spectro-k8s-dashboard.md:177]: @MGamya @arvind-sys 
Please review this AKS section carefully. Are all the necessary steps included? What is the command in step 5 all about? What does it do?

File: content/docs/06-integrations/00-spectro-k8s-dashboard.md
Comment (ritawatson) [inline content/docs/06-integrations/00-spectro-k8s-dashboard.md:97]: @MGamya @arvind-sys 
Are descriptions for **Inherit from Organization** and **None** accurate?

File: content/docs/06-integrations/00-spectro-k8s-dashboard.md
Comment (ritawatson) [inline content/docs/06-integrations/00-spectro-k8s-dashboard.md:221]: @MGamya @arvind-sys 
Is the above example correct?

File: content/docs/06-integrations/00-spectro-k8s-dashboard.md
Comment (ravi-k8) [reply content/docs/06-integrations/00-spectro-k8s-dashboard.md:97]: @ritawatson We don't need to specify the Azure AKS, as we will have different section to list supported clouds to have spectro k8s dashboard integration.

File: content/docs/06-integrations/00-spectro-k8s-dashboard.md
Comment (ravi-k8) [reply content/docs/06-integrations/00-spectro-k8s-dashboard.md]: @ritawatson Yes, label is updated to `Inherit from Organization`

**Comment by spectro-prow** [comment]
@ritawatson: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | b7012af4b6b55968542903e778a5293a6de0595b | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1106/librarium_build/1625227161273110528) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: content/docs/06-integrations/00-spectro-k8s-dashboard.md
Comment (ritawatson) [reply content/docs/06-integrations/00-spectro-k8s-dashboard.md:97]: @ravi-k8 Removed Azure AKS throughout.

File: content/docs/06-integrations/00-spectro-k8s-dashboard.md
Comment (ritawatson) [reply content/docs/06-integrations/00-spectro-k8s-dashboard.md]: Question no longer applies - Azure AKS tab is removed.

File: content/docs/06-integrations/00-spectro-k8s-dashboard.md
Comment (ritawatson) [reply content/docs/06-integrations/00-spectro-k8s-dashboard.md]: Question no longer applies. Azure AKS tab is removed.

File: content/docs/06-integrations/00-spectro-k8s-dashboard.md
Comment (ritawatson) [reply content/docs/06-integrations/00-spectro-k8s-dashboard.md]: Question no longer applies. Azure AKS tab is removed.

File: content/docs/06-integrations/00-spectro-k8s-dashboard.md
Comment (ritawatson) [inline content/docs/06-integrations/00-spectro-k8s-dashboard.md:82]: @ravi-k8 Is this list complete?

File: content/docs/06-integrations/00-kubernetes-dashboard.md
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-kubernetes-dashboard.md:95]:
```suggestion
Starting with Kubernetes Dashboard version 2.7.0, the **Connect** button is no longer available. For an optimized experience, we recommend you to use the Spectro Kubernetes Dashboard integrated pack. 

```

File: content/docs/06-integrations/00-kubernetes-dashboard.md
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-kubernetes-dashboard.md:93]: Let's make this a warning box.

File: content/docs/06-integrations/00-spectro-k8s-dashboard.md
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-spectro-k8s-dashboard.md:20]:
```suggestion
Spectro Kubernetes Dashboard is a web-based UI for Kubernetes clusters that auto-enables the Kubernetes Dashboard. It integrates seamlessly with [Spectro Proxy](/integrations/frp) and uses default settings for access control and Identity Provider (IDP).   
```
Typo, and added a link to the proxy to help the reader.

File: content/docs/06-integrations/00-spectro-k8s-dashboard.md
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-spectro-k8s-dashboard.md:69]:
```suggestion
| ingress.enabled | Ingress configuration to access the ClusterIP, loadBalancer, or nodePort. | `false` |
```
Dot notation is more common to indicate a sub-section of a yaml or JSON object.

File: content/docs/06-integrations/00-spectro-k8s-dashboard.md
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-spectro-k8s-dashboard.md:124]:
```suggestion
1. Copy ``oidc-`` configuration lines in the following code snippet and add them to the Kubernetes pack under the ``extraArgs`` parameter section. Enter your third-party provider details in quotes. <br /><br />
```
I got confused when reading the example and thought I missed something 😵‍💫 . This could just be me, but I think specifying the code snippet will reduce the chances this happening to someone else.

File: content/docs/06-integrations/00-spectro-k8s-dashboard.md
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-spectro-k8s-dashboard.md:169]:
```suggestion
    oidcIdentityProvider:
      identityProviderConfigName: "Spectro-docs" 
      issuerUrl: "issuer-url"    
      usernameClaim: "email"     
      usernamePrefix: "-"      
      groupsClaim: "groups"      
      groupsPrefix: ""                          
      requiredClaims:  ""                          
```
I removed the comments to improve the readability and make it easier for people to copy-paste the code snippet.

**Review by karl-cardenas-coding — APPROVED** [review]
Minor suggestions

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1106#pullrequestreview-1298500423" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1106#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-2/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/06-integrations/00-spectro-k8s-dashboard.md
Comment (ritawatson) [reply content/docs/06-integrations/00-spectro-k8s-dashboard.md]: Fixed.

File: content/docs/06-integrations/00-spectro-k8s-dashboard.md
Comment (ritawatson) [reply content/docs/06-integrations/00-spectro-k8s-dashboard.md]: Fixed typo and added link.

**Review by ravi-k8 — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1106#pullrequestreview-1298500423" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1106#pullrequestreview-1298785048" title="Approved">ravi-k8</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1106#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-2/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by amarjeet27** [comment]
@ritawatson, thanks for putting together this PR, I wanted to share my thoughts on the PR and offer some feedback. I don't think it's a good idea to include the picture that appears in [this preview](https://deploy-preview-1106--docs-spectrocloud.netlify.app/integrations/standalone-integrated-packs), as it was originally intended for an internal conversation and may not accurately represent the details.

Furthermore, introducing the term "integration pack" (or "i-pack") as a separate concept to our customers could be confusing, especially since we already refer to add-on packs as [integrations](https://docs.spectrocloud.com/integrations) in our current documentation. 

However, I do think it would be appropriate to mention that (some of) our packs are better _integrated_ with others (e.g. Spectro Kubernetes Dashboard with Spectro Proxy) and we've simplified configuration and internal dependencies.

cc: @ravi-k8 @prathabk

**Comment by ritawatson** [comment]
@amarjeet27 Thanks for your input. I've made the revisions we discussed.

**Review by amarjeet27 — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1106#pullrequestreview-1300531286" title="Approved">amarjeet27</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1106#pullrequestreview-1298500423" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1106#pullrequestreview-1298785048" title="Approved">ravi-k8</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1106#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-2/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
